smack-integration-test/src/main/java/org/igniterealtime/smack/inttest/util/IntegrationTestRosterUtil.java

```diff
@@ -95,11 +96,21 @@ public static void ensureBothAccountsAreNotInEachOthersRoster(XMPPConnection con
     private static void notInRoster(XMPPConnection c1, XMPPConnection c2) throws NotLoggedInException,
             NoResponseException, XMPPErrorException, NotConnectedException, InterruptedException {
         Roster roster = Roster.getInstanceFor(c1);
+        roster.reloadAndWait();
```

Comment:
That shouldn't be necessary, and if it is, then it's a bug (which we want to uncover with sinttest). Hence this would cover bugs.

Reply:
I'm not sure if I agree fully (a roster update might be in-flight), but it doesn't rule out that after the call, another update is in flight - so it presumably doesn't add much. I have removed the reload.